### PR TITLE
Dataflow configs should not load xml files by default.

### PR DESCRIPTION
**Pull request by sduskis**
Loading the xml files causes look ups in .jar files for .xml files. That causes problems if it's done too often.  Dataflow doesn't have those files anyway, so don't look them up to avoid locking issues found in #630.

### CODE REVIEW

File: bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/BigtableConfiguration.java [hunk 1]
```diff
@@ -44,7 +44,7 @@
   }
 
   public static Configuration configure(String projectId, String zoneName, String clusterName) {
-    Configuration config = new Configuration();
```
Comment (kevinsi4508) [inline bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/BigtableConfiguration.java]: This is not dataflow specific. Are you sure we want to disable loading default?
I don't know the difference, just want to make sure this change is OK.

Comment (sduskis) [inline bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/BigtableConfiguration.java]: A user observed that there are lots of hanging threads in https://github.com/GoogleCloudPlatform/cloud-bigtable-client/issues/630#issuecomment-174023573.  It indicates that new Configuration() tries to find XML files that may be in jars.  That's expensive in terms of time and locking behavior.

Our intended default behavior in dataflow is to have programmatic configuration of the connection and to avoid lookups in xml files, since XML files are harder to set up for this situation.

See https://hadoop.apache.org/docs/r2.6.1/api/org/apache/hadoop/conf/Configuration.html#Configuration(boolean) for more details.

Comment (kevinsi4508) [inline bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/BigtableConfiguration.java]: my question was is this configure() in BigtableConfiguration.java for dataflow only?

Comment (sduskis) [inline bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/BigtableConfiguration.java]: Gotcha.  We needed a programmatic way to configure Dataflow, and this was the way we did it.  It may make sense to discuss additional programmatic idioms for bigtable, but I hope that such discussions are out of scope for this specific change.

Comment (kevinsi4508) [inline bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/BigtableConfiguration.java]: No discussion needed, I just wanted a confirmation from you,

**Comment by kevinsi4508** [comment]
LGTM
